argumentlist.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import "strings"
+
+type argumentList []string
+
+func (l argumentList) String() string {
+	return strings.Join(l, ",")
+}
+
+func (a *argumentList) Set(value string) error {
+	*a = append(*a, strings.Split(value, ",")...)
+	return nil
```

Comment:
It's not a problem for the implementation, empty values are just ignored during processing. So I wouldn't consider it an error.